src/compat/db_json.hpp

```diff
@@ -39,15 +39,19 @@ typedef void JSON_VALIDATOR;
 #include <functional>
 #include "thread.h"
 
+/*
+ * these also double as type precedence
+ * INT and DOUBLE actually have the same precedence
+*/
```

Comment:
Can you shift it to right?

Reply:
I'll remove it altogether because we no longer rely on type precedence.